lib/index.js

```diff
@@ -1,16 +1,89 @@
-'use strict'
-
-var integration = require('@segment/analytics.js-integration')
+'use strict';
 
+var integration = require('@segment/analytics.js-integration');
+var useHttps = require('use-https');
+var reject = require('reject');
 /**
  * Expose `Wigzo` integration
  */
 
 var Wigzo = module.exports = integration('Wigzo')
+  .global('wigzo')
+  .option('orgToken', '')
+  .tag('tracker', '<script src="https://tracker.wigzopush.com/wigzo.js">')
+  .tag('https_manifest', '<link rel="manifest" href="/gcm_manifest.json">')
+  .tag('https_tracker', '<script src="https://tracker.wigzopush.com/wigzopush_manager.js?orgtoken={{ orgToken }}">')
+  .tag('http_tracker', '<script src="https://tracker.wigzopush.com/gcm_http_subscribe.js?orgtoken={{ orgToken }}">');
 
 /**
  * Initialize Wigzo
  */
 
-Wigzo.prototype.initialize = function () {
-}
+Wigzo.prototype.initialize = function() {
+  /* eslint-disable */
+  var orgToken = this.options.orgToken;
+  window.wigzo=function(a){return a.USER_IDENTIFIER="",a.ORGANIZATIONID=orgToken,a.FEATURES={ONSITEPUSH:!0},a}(window.wigzo||{});
+  /* eslint-enable */
+
+  this.load('tracker', this.ready);
+  if (useHttps()) {
```

Comment:
can we write a unit test to cover this `if/else` logic? test coverage is < 100% because this logic is not tested: https://codecov.io/gh/segment-integrations/analytics.js-integration-wigzo/pull/1/diff